tools/scons/site_scons/site_tools/emscripten/emscripten.py

```diff
@@ -8,48 +8,41 @@
 import os
 
 
-def generate(env, emscripten_path=None, **kw):
+def generate(env, **kw):
   """ SCons tool entry point """
 
-  if emscripten_path is None:
-    emscripten_path = os.environ.get('EMSCRIPTEN_ROOT')
-    if not emscripten_path:
-      raise 'Unable to find emscripten. Please set EMSCRIPTEN_ROOT'
-
   # SCons does not by default invoke the compiler with the
-  # environment variabls from the parent calling process,
+  # environment variables from the parent calling process,
   # so manually route all environment variables referenced
   # by Emscripten to the child.
-  for var in ['EM_CACHE', 'EMCC_DEBUG', 'EM_CONFIG',
-              'EMMAKEN_JUST_CONFIGURE', 'EMCC_CFLAGS', 'EMCC_TEMP_DIR',
-              'EMCC_AUTODEBUG', 'EM_COMPILER_WRAPPER',
-              'MOZ_DISABLE_AUTO_SAFE_MODE', 'EMCC_STDERR_FILE',
-              'EMSCRIPTEN_SUPPRESS_USAGE_WARNING', 'NODE_PATH', 'EMCC_JSOPT_MIN_CHUNK_SIZE',
-              'EMCC_JSOPT_MAX_CHUNK_SIZE', 'EMCC_SAVE_OPT_TEMP', 'EMCC_CORES', 'EMCC_NO_OPT_SORT',
-              'EMCC_BUILD_DIR', 'EMCC_DEBUG_SAVE', 'EMCC_SKIP_SANITY_CHECK',
-              'EM_PKG_CONFIG_PATH', 'EMCC_CLOSURE_ARGS', 'JAVA_HEAP_SIZE',
-              'EMCC_FORCE_STDLIBS', 'EMCC_ONLY_FORCED_STDLIBS', 'EM_PORTS', 'IDL_CHECKS', 'IDL_VERBOSE']:
-    if os.environ.get(var):
-      env['ENV'][var] = os.environ.get(var)
-  try:
-    emscPath = emscripten_path.abspath
-  except:
-    emscPath = emscripten_path
-
-  env.Replace(CC=os.path.join(emscPath, "emcc"))
-  env.Replace(CXX=os.path.join(emscPath, "em++"))
+  emscripten_env_vars = ['EM_CACHE', 'EMCC_DEBUG', 'EM_CONFIG',
+    'EMMAKEN_JUST_CONFIGURE', 'EMCC_CFLAGS', 'EMCC_TEMP_DIR',
+    'EMCC_AUTODEBUG', 'EM_COMPILER_WRAPPER', 'MOZ_DISABLE_AUTO_SAFE_MODE',
+    'EMCC_STDERR_FILE', 'EMSCRIPTEN_SUPPRESS_USAGE_WARNING', 'NODE_PATH',
+    'EMCC_JSOPT_MIN_CHUNK_SIZE', 'EMCC_JSOPT_MAX_CHUNK_SIZE',
+    'EMCC_SAVE_OPT_TEMP', 'EMCC_CORES', 'EMCC_NO_OPT_SORT', 'EMCC_BUILD_DIR',
+    'EMCC_DEBUG_SAVE', 'EMCC_SKIP_SANITY_CHECK', 'EM_PKG_CONFIG_PATH',
+    'EMCC_CLOSURE_ARGS', 'JAVA_HEAP_SIZE', 'EMCC_FORCE_STDLIBS',
+    'EMCC_ONLY_FORCED_STDLIBS', 'EM_PORTS', 'IDL_CHECKS', 'IDL_VERBOSE']
+
+  emscons_env_vars = ['PKG_CONFIG_LIBDIR', 'PKG_CONFIG_PATH']
```

Comment:
I think you can/should just add these to the existing to list to keep the diff small.  If you want to create a separate list I guess I would call it `pkg_config_vars` perhaps?